docs/features.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Supported features
+sidebar_label: Features
+---
+
+Margarita is heavily inspired by our <a href="http://kiwi.com" target="_blank">Kiwi.com</a> mobile and web apps. We want to create full-fledged production ready app for developers, entrepreneurs and companies who want to use our API.
+
+## Supported Features
+
+Margarita is separated to 4 main parts of the app:
+
+- [Search for connections](features#search-for-connections)
+- [Booking](features#booking)
+- [Manage my booking](features#manage-my-booking)
+- [Payments](features#payments)
+
+### Search for connections
+
+Get an access to the database of 15 billion connections virtually interlined by <a href="http://kiwi.com" target="_blank">Kiwi.com</a>.
+
+#### Search form
+
+- Initial screen of whole app. User can set here all options about connection he is looking for.
```

Comment:
The initial screen of the whole app. The user can set here all the options about the connection he is looking for, such as the departure date, the place of origin, etc.

Reply:
Done